Source/NSBrowser.m

```diff
@@ -3319,7 +3344,20 @@ - (id) _itemForColumn: (NSInteger)column
 		{
 		  id cell = [selectedCells objectAtIndex: 0];
 
-		  item = [cell objectValue];
+		  if (theBinding != nil)
```

Comment:
I am working on this issue.

Reply:
I believe the better solution here might be to use the column matrix, but I am not certain.  This would eliminate the need for the columnDictionary (possibly) the issue is that the binding based code generally needs the objects to be "nodes" which contain references to children, etc...  the objectValue is used as what is displayed, so setting that the node class wouldn't work right.  I am working through this.

Reply:
I have tried the above.  It seems as though the objectValue needs to be the string or object actually being displayed, not the node.

Reply:
I don't know what you tried and why that failed. Those this mean that in the binding case it is not possible to set the value of the cells, while it works in the other cases? That really sounds weird. I really don't want to block you with this but the current code in this PR seems to have a lot of unnecessary complexity. If that gets into the codebase it is really hard to remove it again. I wouldn't be willing to pay that price but if you want to go ahead with it.

Reply:
The cell of the NSBrowser uses the objectValue as what is displayed in the browser.   What I had tried was to set the objectValue to the actual node.   This results in getting the columns correctly, but the data doesn't display properly.  What I will do is try this another way to see if I can factor out the need for the columnDictionary.